src/config/get-config.mts

```diff
@@ -46,7 +46,12 @@ export async function getConfig(
 	const loadedConfig = await loadConfig<KyselyCTLConfig>({
 		configFile: configArg ? basename(configArg) : undefined,
 		cwd: configPath,
-		dotenv: true,
+		dotenv: {
+			cwd: configPath,
+			fileName: args.environment
+				? ['.env', `.env.${args.environment}`]
+				: '.env',
+		},
```

Comment:
let it fallback to it's default behavior.

```suggestion
		dotenv: {
			cwd: configPath,
			fileName: args.environment
				? ['.env', `.env.${args.environment}`]
				: true,
		},
```

Reply:
@igalklebanov updated with the suggestion 🙌